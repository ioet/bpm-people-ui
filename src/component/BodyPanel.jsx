import React from 'react';
import styles from '../generalStyle.css';


const BodyPanel = () => (
  <div className={styles.bodyPanelStyle} />
);

export default BodyPanel;
