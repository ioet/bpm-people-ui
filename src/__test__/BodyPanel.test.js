import React from 'react';
import BodyPanel from '../component/BodyPanel';

describe('<BodyPanel/>', () => {
  test('Checking default', () => {
    const bodyPanelWrapper = shallow(<BodyPanel />);

    expect(toJson(bodyPanelWrapper)).toMatchSnapshot();
    expect(bodyPanelWrapper.find('div.bodyPanelStyle').length).toEqual(1);
  });
});
