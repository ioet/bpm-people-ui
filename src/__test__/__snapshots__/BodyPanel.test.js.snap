// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BodyPanel/> Checking default 1`] = `
<div
  className="bodyPanelStyle"
/>
`;
